build.gradle

```diff
@@ -24,32 +24,32 @@ ext {
     // Most recent minor versions of all major releases
     // between earliest supported (incl.) and latest stable (excl.), used for binary compatibility checks
     latestMinorsOfOldSupportedMajors: ['2020.1.4', '2020.2.4'],
-    latestStable: '2020.3.2',
+    latestStable: '2020.3.3',
     // Note that we have to use a "fixed snapshot" version X.Y.Z-EAP-SNAPSHOT (e.g. 211.4961.33-EAP-SNAPSHOT)
     // rather than a "rolling snapshot" X-EAP-SNAPSHOT (e.g. 211-EAP-SNAPSHOT)
     // to ensure that the builds are reproducible.
     // EAP-CANDIDATE-SNAPSHOTs can be used for binary compatibility checks,
     // but for some reason aren't resolved in UI tests.
     // Generally, see https://www.jetbrains.com/intellij-repository/snapshots/ -> Ctrl+F idea
-    eapOfLatestSupportedMajor: '211.6693.14-EAP-SNAPSHOT'
+    eapOfLatestSupportedMajor: null //'211.6556.14-EAP-SNAPSHOT'
```

Comment:
**To be restored before this PR is merged**, just trying to get the UI tests to pass...